README.md

```diff
@@ -93,6 +92,8 @@ Flutter Code Editor can detect and fold code blocks. Code blocks folding is supp
 - Python
 - Scala
 
+![foldable_blocks](https://user-images.githubusercontent.com/79095619/213679862-a307333b-d73e-4fc2-99f8-442550e43781.gif)
```

Comment:
The text in `[brackets]` is displayed if the image cannot be loaded. Also screen readers pronounce it for disabled people. So text there should meaningfully explain images.